package.json

```diff
@@ -8,7 +8,9 @@
   },
   "exports": {
     ".": {
-      "require": "./dist/temml.cjs"
+      "require": "./dist/temml.cjs",
+      "import": "./dist/temml.cjs",
```

Comment:
I think this should point to the `mjs` file as `import` typically implies ESM or at least some build system that can handle ESM.